news-bundle/src/EventListener/DataContainer/PageListener.php

```diff
@@ -31,7 +31,7 @@ public function __construct(private Connection $connection, private Security $se
      */
     public function onLoad(DataContainer $dc): void
     {
-        $type = $dc->getCurrentRecord()['type'];
+        $type = $dc->getCurrentRecord()['type'] ?? null;
```

Comment:
If you open the site structure in the back end, there will currently be an error - because `$dc->getCurrentRecord()` will return `null` since there is no record to be fetched yet.